wlm/src/main/java/ai/djl/serving/wlm/LmiConfigRecommender.java

```diff
@@ -107,6 +123,27 @@ private static void setTensorParallelDegree(Properties lmiProperties) {
         lmiProperties.setProperty("option.tensor_parallel_degree", tpDegree);
     }
 
+    private static void setDynamicBatch(
+            Properties lmiProperties,
+            LmiUtils.HuggingFaceModelConfig modelConfig,
+            ModelInfo<?, ?> modelInfo,
+            String features) {
+        // TODO TrtLLM python backend: Change it once TrtLLM supports T5 with inflight batching.
+        if ("t5".equals(modelConfig.getModelType())
```

Comment:
nit: we're using this same condition in a couple places, maybe move to a single method `isT5TrtLlm`

Reply:
Done